### PR TITLE
update background@2x image

### PR DESCRIPTION
**Pull request by Kallecrafter**
## Type of change

- [ ] Add new directory for a server
- [x] Update directory of a server
- [ ] Remove directory of a server
- [x] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

**Comment by Nils-afk** [comment]
The new Image has the wrong size. Yours is W: 1920px | H: 1020px. It should be W: 1920px | H: 1080px Than Convert it and it should work!

**Comment by Kallecrafter** [comment]
craftergang.de it's not Offline!
image

**Comment by maaaathis** [comment]
> Its Online! image Please Fix the Bot Problem!

This is an external provider calling your server. For this reason, the action says “[...] could be offline” and not “is offline”. However, this is absolutely no problem if your server is really online.

File: minecraft_servers/craftergang/background@2x.png
Comment (DoJapHD) [inline minecraft_servers/craftergang/background@2x.png:1]: Compress image

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Comment by Kallecrafter** [comment]
Image was compressed

**Comment by Kallecrafter** [comment]
Now everything is correct. Now you can merge (:

**Review by DoJapHD — APPROVED** [review]

**Comment by Kallecrafter** [comment]
closed

**Pull request merged**